### ISSUE TITLE
Every Data Backup Ability

### ISSUE THREAD

**Issue by IzumiSenaSora** [amnesica/KryptEY#23]
If i uninstall keyboard i need to do everything from scratch and add people in my keyboard... It's a pain... Can you add ability to backup every data like keyboard settings and messages history and added people keys and name... All possible data ... So i can export and import when needed ...

**Comment by harishbewe** [comment]
I too felt the same. Thanks for posting this

**Comment by serrq** [comment]
Yes, but with modular backup design. One can backups just what it needs. Not necessarily the whole package (screenshot taken from Libretube app).

![modular-backup](https://github.com/amnesica/KryptEY/assets/113377431/d7f43ed5-5c85-4e4d-bbaf-7540558237a9)
